apps/patient/src/utils/text.tsx

```diff
@@ -32,6 +32,12 @@ export const text = {
   open: 'Open',
   open24hrs: 'Open 24 Hours',
   orderCanceled: 'This order was canceled.',
+  orderCanceledHeader: 'Your order was canceled',
```

Comment:
i tend to like to exclude denoting elements like "header", "subheader" since I might use the text in multiple places, just preference

Reply:
yup makes sense, updating